### PR TITLE
Added notebook with model using scaled features

### PR DESCRIPTION
**Pull request by tm-jace-peralta**
This PR adds the notebooks used to enhance cross-country model performance
1. `2023-02-06_crosscountry_initial_jace.ipynb` - experimented on different scalers applied on the feature columns and wealth index. Performance was increased (R^2 = 0.41 -> 0.48) especially for the case where PH is the test country (R^2 = 0.25->0.42).

### CODE REVIEW

**Review by alronlam — CHANGES_REQUESTED** [review]
Please see comments from ReviewNB

**Comment by alronlam** [comment]
Oh one last thing, can you add a bit of discussion at the top and refer to the 2 reference papers we found that also normalized features and wealth index per country? Just so we remember the basis for doing something like this. Thanks!

File: notebooks/2023-01-24-dhs-cross-country-experiments/2023-02-06_crosscountry_normalized.ipynb [hunk 1]
```diff
@@ -0,0 +1,1476 @@
+{
```
Comment (alronlam) [inline notebooks/2023-01-24-dhs-cross-country-experiments/2023-02-06_crosscountry_normalized.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #51.    print(f'Mean split r^2  (std): {split_r2_mean} ({split_r2_std})')
</pre><p>Last thing, can we please update the code here to reflect the best results/recommendations you have? </p><p> </p><p>You mentioned MinMaxScaler for features, then StandardScaler for the Wealth Index. These results and scatterplots seem to be different from the ones you previously reported. Thanks!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/unicef-ai4d-poverty-mapping/pull/117/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Comment (tm-jace-peralta) [reply notebooks/2023-01-24-dhs-cross-country-experiments/2023-02-06_crosscountry_normalized.ipynb:1]: Notebook scaling function and cells are now fixed, and now displays best config: per-country `StandardScaler` for both feature and label columns.

**Review by alronlam — APPROVED** [review]

**Pull request merged**